readme.md

```diff
@@ -16,7 +16,12 @@ the square brackets (for multiple-choice questions).
 Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
-```js
+```$.ajax({
+  url: http://kittengifs.com/gifs,
+  data: [],
```

Comment:
no need for `data` since this should be a "GET" request. Also, you should probably specify the method as GET explicitly.

Finally, you're referencing a very old way of calling AJAX, which includes a `success` function, which currently isn't defined. The more modern way (that we taught in class) is to use `.done(function() { ... })` to register callbacks on success.